### PR TITLE
Added newest version of headless AI

### PR DESCRIPTION
**Pull request by jimmierustler**
Changes:
- Stances fixed.
- Some new settings.
- BI Waypoint Bug resolved.
- Improved performance.

### CODE REVIEW

File: description.ext [hunk 1]
```diff
@@ -1,17 +1,20 @@
 author = "AUTHOR NAME";
-OnLoadName = "Operation Name";
-OnLoadMission = "Loading text";
+OnLoadName = "MISSION NAME";
+OnLoadMission = "MISSION DESCRIPTION";
 loadScreen = "load.jpg";
 
+enableDebugConsole[] = {"76561199020930438"};
```
Comment (gkovalechyn) [inline description.ext]: Whose Id is this?

File: description.ext [hunk 2]
```diff
@@ -1,17 +1,20 @@
...
 
+enableDebugConsole[] = {"76561199020930438"};
+
 // respawnTemplates[] = {"TCS_OneLife"}; // One life respawn
-// respawnTemplates[] = {"TCS_Respawn", "TCS_Tickets"}; // Ticket based respawn
-// respawnTemplates[] = {"TCS_Respawn", "TCS_Individual_Tickets"}; // Individual ticket based respawn
-respawnTemplates[] = {"TCS_Respawn", "Counter"}; // Default respawn
+ respawnTemplates[] = {"TCS_Respawn", "TCS_Tickets"}; // Ticket based respawn
+//respawnTemplates[] = {"TCS_Respawn", "TCS_Individual_Tickets"}; // Individual ticket based respawn
+// respawnTemplates[] = {"TCS_Respawn", "Counter"}; // Default respawn
```
Comment (gkovalechyn) [inline description.ext]: Do we want to change the default option?

File: description.ext (hunk 1 above)
Comment (jimmierustler) [reply description.ext]: oh... uhhh... whoops

File: description.ext (hunk 2 above)
Comment (jimmierustler) [reply description.ext]: No looks like my personal mission got added to source control. I'll fix that

**Review by jimmierustler — COMMENTED** [review]
Changes:

Stances fixed.
Some new settings.
BI Waypoint Bug resolved.
Improved performance.

**Comment by jimmierustler** [comment]
This is cursed

**Pull request closed without merging**